integreat_cms/core/settings.py

```diff
@@ -291,6 +291,17 @@
     os.environ.get("INTEGREAT_CMS_TEXTLAB_API_DEFAULT_BENCHMARK_ID", 420)
 )
 
+#: List of categories for which we want to process HIX feedback details provided by Textlab API
+TEXTLAB_API_FEEDBACK_CATEGORIES: Final[list[str]] = [
+    "moreSentencesInClauses",
+    "moreSentencesInWords",
+    "moreWordsInLetters",
+    "countPassiveVoiceInSentence",
+    "countInfinitiveConstructions",
+    "countNominalStyle",
+    "countFutureTenseInSentence",
```

Comment:
I know these names come from the textlab api, but could you add some comment explaining their meaning?
I don't really understand what e.g. `moreSentencesInWords` is supposed to mean.

Also I am not really sure if the settings are the best place for this. For example if we wanted to add a new category, it would not be sufficient to only update the settings. We would still have to make a new release where the mapping in `static/src/js/analytics/hix-widget.ts` is updated.
If you decide not to use the settings, maybe you could have a mapping in the textlab api client?

Roughly like this:
```python
hix_feedback_mapping = {
    "moreSentencesInClauses": "hix-count-nested-sentences",
    "moreSentencesInWords": "hix-count-long-sentences",
    "moreWordsInLetters": "hix-count-long-words",
    # ...etc
}
feedback_details = {
    cms_name: response.get(api_name, [])
    for api_name, cms_name in hix_feedback_mapping.items()
}
```
Then you could get rid of the mapping in the typescript code and directly use these values

Reply:
> For example if we wanted to add a new category, it would not be sufficient to only update the settings

True, but now we can remove categories without updating the code. 
No idea if this could be a useful though 🙈

Reply:
I don't have a clear opinion on where to put these categories. I'll try asking on the channel, maybe there are more opinions on this there.

Reply:
> Then you could get rid of the mapping in the typescript code and directly use these values

Maybe that's not bad, I will think of it next week.

Reply:
Moved the mapping to the textlab api client
Revised feedback json structure to match standard json style (property names in camel case)